.travis.yml

```diff
@@ -42,6 +42,9 @@ before_install:
     esac
 
     pip install pylint
+    pip install futures==3.1.1
```

Comment:
Done. It is actually a dependency of grpc_debug_server, which comes from the tensorflow package and is used by the debugger plugin. See:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/debug/lib/grpc_debug_server.py#L26